sensor/kubernetes/client/client.go

```diff
@@ -77,6 +80,17 @@ func mustCreateOpenshiftConfigClient(config *rest.Config) configVersioned.Interf
 	return client
 }
 
+func mustCreateOpenshiftOperatorClient(config *rest.Config) operatorVersioned.Interface {
+	if !env.OpenshiftAPI.BooleanSetting() {
+		return nil
+	}
+	client, err := operatorVersioned.NewForConfig(config)
+	if err != nil {
+		log.Warnf("Could not generate openshift operator client: %s", err)
```

Comment:
This should be `%v`, no? I don't recall if `.String()` is automatically called, but I know in our codebase we have usually used `%v` for errors

Reply:
I agree that `%v` is usually used and makes more sense, however the other instances in this file that call `log.Warnf()` were using `%s` so I was keeping it consistent (and yes it's valid to pass an error in place of `%s`).

I'll change it to `%v` - perhaps the 'higher' level of consistency makes sense long term.